services/chainstorage/api/src/main/java/tech/pegasys/teku/services/chainstorage/api/StorageConfiguration.java

```diff
@@ -104,14 +119,16 @@ public static final class Builder {
 
     private Eth1Address eth1DepositContract;
     private StateStorageMode dataStorageMode = StateStorageMode.DEFAULT_MODE;
-    private long dataStorageFrequency = VersionedDatabaseFactory.DEFAULT_STORAGE_FREQUENCY;
+    private long dataStorageFrequency = 2048L;
```

Comment:
This probably should use a constant even if the constant winds up migrating to this class.  The value should be the same here and in `BeaconNodeDataOptions` so both should point to the same constant.

Reply:
hrm i'd forgotten about that bit, I was breaking a cyclic dependency, can look at making that cleaner.